src/style.css

```diff
@@ -0,0 +1,143 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use any rule inside `*` selector instead of `box-sizing` rule.
margin and padding you have to use inside selectors where you definitely need it